app/components/tabs.tsx

```diff
@@ -0,0 +1,36 @@
+import { Tab } from "@headlessui/react";
+import { Fragment } from "react";
+
+type Props = {
```

Comment:
I think instead of using props it would be better to use dot notation. So we might have something that looks more like [this](https://mantine.dev/core/tabs/)

Reply:
@bcinman thoughts on dot notation vs props here?

Reply:
I'm happy to go any direction on this, just thought maybe since we use this component a lot we could try abstracting it out

Reply:
I get where you're going with this @thehannahlerman. 

I think we did something similar in the old flipside-app. I've generally come prefer component composition over configuration. Which means having separate components that can be used together, rather than having a prop.

Agree with @sebruiz here.

Reply:
def helpful to discuss our practices now so we can keep it consistent in the whole app